amd/src/exporter.js

```diff
@@ -18,7 +18,7 @@ export default class {
         }
 
         let showactionmenu = state.common.userboards == 1 || state.common.groupselector != '' ||
-            state.capabilities.get('manageboards').value ||
+            state.capabilities.get('manageboard').value ||
```

Comment:
Errors like this make me think if it would be a good idea to avoid magic strings like these and define all the necessary capability keys used in the JS frontend in a separate file containing an object

```
capabilities = {
  MANAGEBOARD: 'manageboard',
  VIEWHISTORY: 'viewhistory',
  // TODO add all the remaining capabilities
};
```
This file could be imported whereever capabilities are accessed from the frontend JS. This would also help when developing, because it will be immediately clear what capabilities exist.

Thoughts?